src/platform/site-wide/announcements/components/PrePreDowntime.jsx

```diff
@@ -0,0 +1,41 @@
+// Dependencies.
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import PromoBanner, {
+  PROMO_BANNER_TYPES,
+} from '@department-of-veterans-affairs/formation-react/PromoBanner';
+
+class PrePreDowntime extends Component {
+  static propTypes = {
+    announcement: PropTypes.shape({
+      downtimeStartsAt: PropTypes.string.isRequired,
+      downtimeExpiresAt: PropTypes.string.isRequired,
+    }).isRequired,
+    dismiss: PropTypes.func.isRequired,
+  };
+
+  render() {
+    const {
+      announcement: { downtimeStartsAt, downtimeExpiresAt },
+      dismiss,
+    } = this.props;
+
+    // Derive the message.
+    const formattedStartsAt = moment(downtimeStartsAt).format(
+      'MMM Do [at] h:mm a',
+    );
+    const formattedExpiresAt = moment(downtimeExpiresAt).format('h:mm a z');
+    const message = `We'll be doing site maintenance on ${formattedStartsAt} until ${formattedExpiresAt}. You won’t be able to sign in or use some tools during this time.`;
+
+    return (
+      <PromoBanner
+        onClose={dismiss}
+        render={() => <div>{message}</div>}
```

Comment:
Ah, I forgot that we have to use a custom `render` because otherwise PromoBanners expect an `HREF` value.

Reply:
Just to confirm, it's ok if an `href` prop isn't provided? I didn't get any `PropType` errors from it

Reply:
Yep, you're good! https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation-react/src/components/PromoBanner/PromoBanner.jsx#L68